web/local-router.php

```diff
@@ -2,10 +2,9 @@
 if (preg_match('#^/assets/#', $_SERVER["REQUEST_URI"])) {
     // always pass through for static assets
     return false;
-} elseif (preg_match('/\\.pdf$/', $_SERVER["REQUEST_URI"])) {
-    // for non-asset PDF URLs, explicitly send to index.php controller to avoid
+} elseif (preg_match('/\\.pdf|csv|json$/', $_SERVER["REQUEST_URI"])) {
+    // for non-asset PDF, CSV and JSON URLs, explicitly send to index.php controller to avoid
```

Comment:
Guessing this is riding along with this change, but not specifically related.

Reply:
Yes, not specifically related but need to change this in order to test csv exports in localhost.